### PR TITLE
Add checker for allow_null attr

### PR DESCRIPTION
**Pull request by krectra**
fixes for error occurring  on serpy serializers

### CODE REVIEW

File: src/drf_yasg/inspectors/base.py [hunk 1]
```diff
@@ -281,8 +281,10 @@ def SwaggerType(existing_object=None, use_field_title=True, **instance_kwargs):
                 instance_kwargs.setdefault('title', title)
             if description is not None:
                 instance_kwargs.setdefault('description', description)
-            if field.allow_null:
-                instance_kwargs['x_nullable'] = True
+            
+            if hasattr(field, 'allow_null'):
+                if field.allow_null:
```
Comment (d3QUone) [inline src/drf_yasg/inspectors/base.py:286]: you can simplify these two ifs:
```python
if getattr(field, 'allow_null', None):
    instance_kwargs['x_nullable'] = True
```

**Comment by EralpBay** [comment]
Please merge this :+1: it's basically not usable with Serpy Serializers. https://github.com/axnsan12/drf-yasg/issues/687

File: src/drf_yasg/inspectors/base.py (hunk 1 above)
Comment (krectra) [reply src/drf_yasg/inspectors/base.py]: Updated. thanks.

**Comment by derneuere** [comment]
Hey, would it be possible to merge this pull request or are there any other issues that need to be fixed?

**Review by d3QUone — APPROVED** [review]

**Comment by d3QUone** [comment]
I think yes, it's fine. But only project collaborators have access to merge..

**Comment by rlrojasm** [comment]
Hi It's still pending to merge @krectra  could you take a look at it!. Thanks

**Comment by krectra** [comment]
> Hi It's still pending to merge @krectra could you take a look at it!. Thanks

Pending workflow approval

**Comment by sabyashaikh** [comment]
Hello, any update on this issue?

**Pull request merged**